.ci/pitest.sh

```diff
@@ -150,6 +150,7 @@ pitest-javadoc)
   "AbstractJavadocCheck.java.html:<td class='covered'><pre><span  class='survived'>        beginJavadocTree(root);</span></pre></td></tr>"
   "AbstractJavadocCheck.java.html:<td class='covered'><pre><span  class='survived'>        finishJavadocTree(root);</span></pre></td></tr>"
   "AbstractJavadocCheck.java.html:<td class='covered'><pre><span  class='survived'>        javadocTokens.clear();</span></pre></td></tr>"
+  "JavadocParamOrderCheck.java.html:<td class='covered'><pre><span  class='survived'>       if (typeDef.getType() == type) {</span></pre></td></tr>"
```

Comment:
We will not merge PR's that add to our technical debt. Remove this suppression.

Reply:
removed this line from pitest.sh

Reply:
Resolved.